snscrape/modules/twitter.py

```diff
@@ -1805,7 +1835,6 @@ def get_items(self):
 			'verified_phone_label_enabled': False,
 			'responsive_web_graphql_timeline_navigation_enabled': True,
 			'responsive_web_graphql_skip_user_profile_image_extensions_enabled': False,
-			'responsive_web_graphql_skip_user_profile_image_extensions_enabled': False,
```

Comment:
what does removing this do?

Reply:
It was a duplicate of the line above it. Figured I would yeet it while I was in there.

Reply:
Oh, yes, I didn't see that!